rel/value_repr_test.go

```diff
@@ -0,0 +1,24 @@
+package rel
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestRepr(t *testing.T) {
```

Comment:
More tests? seems pretty light considering how many types just got a repr function.
Most importantly, it seems tuples are completely missing (sets, strings, arrays, dicts, and some nested elements thereof are tested in one way or another).